### PR TITLE
strip: add page

### PR DESCRIPTION
**Pull request by matteo-briani**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/strip.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# strip
+
+> Discard symbols from executables or object files.
```
Comment (mfrw) [inline pages/linux/strip.md:3]:
```suggestion
> Discard symbols from executables or object files.
> More information: <https://manned.org/strip>.
```

Comment (matteo-briani) [reply pages/linux/strip.md:3]: thanks for the add!

File: pages/linux/strip.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://manned.org/strip>.
+
+- Replace the input file with its stripped version:
+
+`strip {{/path/to/file}}`
```
Comment (pixelcmtd) [inline pages/linux/strip.md:8]:
```suggestion
`strip {{path/to/file}}`
```

File: pages/linux/strip.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Put the stripped output in file, rather than replacing the existing one:
+
+`strip {{/path/to/input_file}} -o {{/path/to/output_file}}`
```
Comment (pixelcmtd) [inline pages/linux/strip.md:12]:
```suggestion
`strip {{path/to/input_file}} -o {{path/to/output_file}}`
```

File: pages/linux/strip.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Strip debug symbols only:
+
+`strip --strip-debug {{/path/to/file.o}}`
```
Comment (pixelcmtd) [inline pages/linux/strip.md:16]:
```suggestion
`strip --strip-debug {{path/to/file.o}}`
```

**Review by mfrw — APPROVED** [review]
@matteo-briani Thank you for your contribution.
LGTM

File: pages/linux/strip.md (hunk 4 above)
Comment (matteo-briani) [reply pages/linux/strip.md]: thanks for the improvements

File: pages/linux/strip.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# strip
+
+> Discard symbols from executables or object files.
+> More information: <https://manned.org/strip>.
+
+- Replace the input file with its stripped version:
+
+`strip {{path/to/file}}`
+
+- Put the stripped output in file, rather than replacing the existing one:
```
Comment (navarroaxel) [inline pages/linux/strip.md:10]: I think this is more explicit and cleaner:
```suggestion
- Strip symbols from a file, saving the output to a specific file:
```

Comment (matteo-briani) [reply pages/linux/strip.md:10]: agree

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**